DataFormats/PatCandidates/interface/PATObject.h

```diff
@@ -768,6 +770,24 @@ namespace pat {
   }
 
   template <class ObjectType>
+  void PATObject<ObjectType>::addUserDataObject_( const std::string & label, std::auto_ptr<pat::UserData> & data, bool overwrite ) 
```

Comment:
Most of our code, including `OwnVector` still uses `auto_ptr`.
That said, yes, I can move it to `unique_ptr`